packages/worker/src/api/controllers/global/tenant.ts

```diff
@@ -0,0 +1,6 @@
+import { tenancy } from "@budibase/backend-core"
+import { TenantInfo, Ctx } from "@budibase/types"
+
+export const save = async (ctx: Ctx<TenantInfo>) => {
+  await tenancy.saveTenantInfo(ctx.request.body)
```

Comment:
I would usually recommend returning something in the body - e.g. the `_id` of the document, and a message to say saved successfully.

Reply:
returning body with _id